repo/fsrepo/datastores.go

```diff
@@ -410,3 +412,36 @@ func (c *badgerdsDatastoreConfig) Create(path string) (repo.Datastore, error) {
 
 	return badgerds.NewDatastore(p, &defopts)
 }
+
+// PostgresDatastoreConfig returns a postgres DatastoreConfig from a spec
+func PostgresDatastoreConfig(params map[string]interface{}) (DatastoreConfig, error) {
+	var c postgresDatastoreConfig
+	var ok bool
+	c.path, ok = params["path"].(string)
+	if !ok {
+		log.Fatal("no path")
+	}
+
+	return &c, nil
+}
+
+type postgresDatastoreConfig struct {
+	path string
+}
+
+func (c *postgresDatastoreConfig) DiskSpec() DiskSpec {
+	return map[string]interface{}{
+		"type": "postgresds",
+		"path": c.path,
```

Comment:
`path` is not needed here, but I'm not sure what to put here that makes sense. My guess would be user+database, as host/password may change frequently in some setups.